runelite-client/src/main/java/net/runelite/client/plugins/timetracking/TimeTrackingPlugin.java

```diff
@@ -219,6 +218,24 @@ public void checkCompletion()
 		}
 	}
 
+	@Schedule(period = 500, unit = ChronoUnit.MILLIS)
+	public void checkTimerOrder()
+	{
+		boolean timerOrderChanged = clockManager.checkTimerOrder();
+
+		if (timerOrderChanged)
+		{
+			panel.update();
+		}
+	}
+
+	@Schedule(period = 500, unit = ChronoUnit.MILLIS)
+	public void checkTimerWarning()
```

Comment:
Same here, and also you dont need 2 scheduled methods anyway

Reply:
done